.github/workflows/run_mmperf.yml

```diff
@@ -17,6 +17,7 @@ on:
   schedule:
     - cron: '0 13 * * *'
   workflow_dispatch:
+  pull_request:
```

Comment:
Remove `pull_request` before merge. Also reminder that you might be able to use `workflow_dispatch` to test PRs like this without fiddling with git.

1. push to a branch in the main repo (not a fork)
2. go to https://github.com/openxla/iree/actions/workflows/run_mmperf.yml
3. run workflow with trigger on that branch: 
![image](https://user-images.githubusercontent.com/4010439/234049118-a69e97e3-caed-4e2b-a8ac-77dc8fb22f5f.png)

(and if you do that, you could add `skip-ci: only changing mmperf` to the PR description so PRs like this skip the regular CI builds - CI minutes are somewhat pricy...)